requirements.txt

```diff
@@ -1,4 +1,4 @@
-bleak==0.21.1
+bleak==0.22.3
```

Comment:
`pyproject.toml` is the file actually being used for declaring package dependencies, there is a case to be made for adding version restrictions to it but I don't think they should be pinned necessarily. 

`requirements.txt` is vestigial, or at best a convenience for setting up a venv when deving on pyanura

This is maybe somewhat confusing. I'm not really sure what a

Reply:
Got it. It is possible to install dependencies from `pyproject.toml` by doing `pip install .[cli]` in the root folder of the repo. What about moving all dependencies to `pyproject.toml`? I see that there are some dependencies here in this file that are not present in `pyproject.toml` - are they needed? I seem to be able to run the CLI using only `pip install .[cli]` and not installing dependencies from `requirements.txt`.

Reply:
I added a commit removing requirements.txt. Apparently it was used in CI as well. I changed that to an approach that uses pyproject.toml instead. It seems to still be working. What do you think?